cli/cmd/graphenePrepare_test.go

```diff
@@ -142,5 +142,5 @@ func TestDownloadPremain(t *testing.T) {
 
 	// We should have one download here
 	info := httpmock.GetCallCountInfo()
-	assert.Equal(1, info[`GET =~^https://github\.com/edgelesssys/marblerun/releases/download/v[0-9\.]*/premain-graphene`])
+	assert.Equal(1, info[`GET =~^https://github\.com/edgelesssys/marblerun/releases/download/.*/premain-graphene`])
```

Comment:
Mhh, I don't know about `dev-build`.

Personally, having the version encoded in a sane way allows us to unit test download URLs or anything where we dynamically parse the version into.

Reply:
> Mhh, I don't know about `dev-build`.

I don't have any strong opinion here. It is a placeholder value for a version string that is replaced at build time.
So I'm open to suggestions. I see @daniel-weisse's point that `v0.0.0-dev` could confuse people. But if there is no better option though, I'm fine with `v0.0.0` or `v0.0.0-dev` as well. It's not that big of a deal IMO.

> Personally, having the version encoded in a sane way allows us to unit test download URLs or anything where we dynamically parse the version into.

Sure. Ideally, that's the case. 

> This change here basically breaks the test. It's intend was to check if it sets a plausible URL, now it straight out just accepts every sequence of characters, which is not great and makes the test kinda obsolete.

It accepts every type of version string, the URL is still intact. We're basically testing the parsing of placeholder value here, so I could live with that.

If there is no objection from @daniel-weisse I'll go with the `v0.0.0` string and add a comment.

Reply:
Sounds good to me